spec/datadog/tracing/contrib/span_attribute_schema_examples.rb

```diff
@@ -13,13 +13,17 @@
     context 'test the default' do
       it do
         expect(span.service).to eq('rspec')
+        span.set_tag('_expected_service_name', 'rspec')
+        span.set_tag('_remove_integration_service_names_enabled', 'true')
```

Comment:
Could you explain what's happening here?

Reply:
I updated this file to include a comment. Here we are grabbing the expected service name and putting it in the span meta in order to later send it to the test agent for service name testing to ensure consistency for service name when the `_remove_integration_service_names_enabled ` is set to true.

Reply:
Ok, I understand so far.

When should I use these two tags myself?